core/db_models/fields/EE_JSON_Field.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+use EventEspresso\core\services\json\JsonDataHandler;
+
+class EE_JSON_Field extends EE_Model_Field_Base
+{
```

Comment:
had to add this class because it is used by `EEM_Form_Submission` for the form data